resources/views/themes/default1/order/show.blade.php

```diff
@@ -746,5 +743,26 @@ function checking(e){
         });
 
 
+    document.querySelectorAll('span[data-type="copy"]')
```

Comment:
this script is seen multiple times in this PR. We could have tried some way of reusing it

Reply:
Moved under common/js folder